GitCommands/Git/Executable.cs

```diff
@@ -150,8 +150,11 @@ private void OnProcessExit(object sender, EventArgs eventArgs)
                             if (_throwOnErrorExit && exitCode != 0)
                             {
                                 string errorOutput = _process.StandardError.ReadToEnd().Trim();
-                                ExternalOperationException ex
-                                    = new(command: _process.StartInfo.FileName,
+
+                                Exception ex
+                                    = GitDubiousOwnershipSecurityException.IsGitDubiousOwnershipSecurityError(errorOutput)
+                                ? new GitDubiousOwnershipSecurityException(errorOutput)
+                                : new ExternalOperationException(command: _process.StartInfo.FileName,
```

Comment:
This looks misindented
```suggestion
                                Exception ex = GitDubiousOwnershipSecurityException.IsGitDubiousOwnershipSecurityError(errorOutput)
                                    ? new GitDubiousOwnershipSecurityException(errorOutput)
                                    : new ExternalOperationException(command: _process.StartInfo.FileName,
```